README.Rmd

```diff
@@ -104,6 +104,26 @@ b <- data.frame(specimenID = LETTERS[1:4])
 result$data
 ```
 
+# Data submission validation
+
+This package contains a Shiny app to validate manifests and metadata for AMP-AD
+studies. It uses the
+[dccvalidator](https://github.com/Sage-Bionetworks/dccvalidator) package to
+check for common data quality issues and gives realtime feedback to the data
+contributor on errors that need to be fixed. The reporting UI is heavily inspired by the 
+[MetaDIG project's metadata quality reports](https://knb.ecoinformatics.org/quality/s=knb.suite.1/doi%3A10.5063%2FF12V2D1V).
+
+## Deployment
+
+The app is deployed on the Sage Bionetworks' Shiny Pro server through the following steps:
+
+1. ssh into the Shiny Pro server and navigate to `/home/kwoo/ShinyApps/dccvalidator-app`
```

Comment:
If you don't think it is strange to link a public repo to a private Confluence page, I do think it could be useful to others at Sage hoping to use this app.

Reply:
I'll add a note that these are instructions for people at Sage deploying the app

Reply:
@kelshmo I added some additional instructions, let me know if this is clearer.

Reply:
thank you!